packages/lib-classifier/src/store/utils/session.js

```diff
@@ -1,19 +1,12 @@
 import hash from 'hash.js'
 
-const storage = window.sessionStorage || window.localStorage
```

Comment:
I'm not sure why the option for localStorage is here. This feature is supposed to be a session id, which refreshes just like a browser session. If researchers are using session ids to analyze anonymous users, then don't use localStorage as a backup.

Reply:
1. ✔️ Agreed, in this scenario a fallback is actually a bad option that leads to inconsistent/unpredictable behaviour. Removing localStorage is a good idea.
2. ❓ I'm wondering if the rest of the session.js needs to be made more robust. I just realised a lot of the code in this file does things like `window.sessionStorage` and `storage.getItem()`, instead of `window?.sessionStorage` and `storage?.getItem()`, which may inexplicably crash the process if window or window.sessionStorage is ever unavailable. (Question for Shaun: when would such a scenario happen? Browser security settings? Incognito mode? "Do not track" enabled?)

Reply:
Because the classifier runs only in the browser, the chance of `window` being undefined is lower than components of app-project, for example, and the entire classifier is also lazy loaded. I'm not sure if user browser settings could impact `window` 🤔  In a test env we've mocked `window`.

Reply:
`sessionStorage` will throw a `SecurityException` if it violates browser privacy settings, so you should catch that. 

https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage#exceptions

Reply:
You might be able to throw that error by setting the browser to block `www.zooniverse.org` in an ad blocker or in cookie preferences.

Reply:
Good point!